tyk-headless/tests/deployment-gw-repset_test.yaml

```diff
@@ -0,0 +1,294 @@
+suite: test headless gateway deployment
+templates:
+  - templates/configmap-gateway.yaml
+  - templates/deployment-gw-repset.yaml
+  - templates/secrets.yaml
+  - templates/secret-certs.yaml
+release:
+  name: my-release
+  namespace: my-namespace
+chart:
+  version: 0.12.0
+tests:
+  - it: should match metadata name with default values
+    template: templates/deployment-gw-repset.yaml
+    asserts:
+      - equal:
```

Comment:
Instead of `equal` should we use `isAPIVersion` assertion like you did below?

Reply:
yes, we can use `isAPIVersion`. I will update the assertion accordingly. Thank you 🙏